src/main/java/org/hobbit/core/containerservice/DirectContainerCreator.java

```diff
@@ -0,0 +1,195 @@
+package org.hobbit.core.containerservice;
+
+import java.nio.ByteBuffer;
+import java.util.Arrays;
+import java.util.Set;
+import java.util.UUID;
+import java.util.concurrent.Future;
+import org.apache.commons.io.Charsets;
+import org.hobbit.core.Commands;
+import org.hobbit.core.Constants;
+import org.hobbit.core.components.AbstractBenchmarkController;
+import org.hobbit.core.components.AbstractCommandReceivingComponent;
+import org.hobbit.core.data.StartCommandData;
+import org.hobbit.core.rabbit.RabbitMQUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import com.google.common.util.concurrent.SettableFuture;
+import com.rabbitmq.client.AMQP.BasicProperties;
+
+/**
+ * This class provides the implementation to create container functionality
+ * @author altaf, sourabh, yamini, melisa
+ *
+ */
+public class DirectContainerCreator implements ContainerCreation {
+	
+	private static final Logger LOGGER = LoggerFactory.getLogger(DirectContainerCreator.class);
+	
+	AbstractCommandReceivingComponent directComponent = null;
+	
+	private AbstractBenchmarkController abstractBenchmarkController;
```

Comment:
Why do you need a benchmark controller? This shouldn't be necessary. What happens, if the system adapter would like to create a new container? What happens, if a data generator needs to create a new container?